packages/frontend/src/components/Explorer/ExploreTree/TableTree/Tree/TreeSingleNode.tsx

```diff
@@ -51,10 +57,19 @@ const TreeSingleNode: FC<Props> = ({ node }) => {
 
     const isFiltered = isField(item) && isFilteredField(item);
 
-    const label = timeIntervalLabel || item.label || item.name;
-
+    const label = isField(item) ? timeIntervalLabel || item.label : item.name;
+    const description = isField(item) ? item.description : undefined;
     const bgColor = getItemBgColor(item);
 
+    const getFieldIconColor = (field: Item | AdditionalMetric) => {
+        if (isCustomDimension(field) || isDimension(field)) return 'blue.9';
```

Comment:
Potential things to refactor, 

Instead of doing all these `if is type` methods, we could have a function (getFieldType) that return sa string, that we can use on a `switch`